src/app/auth/components/signup/signup.component.scss

```diff
@@ -119,3 +119,24 @@ form {
   display: flex;
   gap: 16px;
 }
+
+.strength-too-weak {
+  color: red;
+}
+
+.strength-weak {
+  color: orange;
+}
+
+.strength-medium {
+  color: blue;
+}
+
+.strength-strong {
+  color: green;
```

Comment:
This is not an request for changes immediately but we probably want some design/accessibility review here for color choices that match Permanent's theme and also have proper contrast against the background.